var/spack/repos/builtin/packages/py-fenics-ufl-legacy/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyFenicsUflLegacy(PythonPackage):
+    """The Unified Form Language (UFL) is a domain specific language for
+    declaration of finite element discretizations of variational forms. More
+    precisely, it defines a flexible interface for choosing finite element
+    spaces and defining expressions for weak forms in a notation close to
+    mathematical notation."""
+
+    homepage = "https://fenicsproject.org/"
+    url = "https://github.com/FEniCS/ufl-legacy/archive/2022.3.0.tar.gz"
+    git = "https://github.com/FEniCS/ufl-legacy.git"
+    maintainers("chrisrichardson", "garth-wells", "jhale")
+
+    license("LGPL-3.0-or-later")
+
+    version("main", branch="main")
+    version("2022.3.0", sha256="2d0f4c88fe151d631e1d389faf61f58bbbe649fd08106e756fd5d6c53213660a")
+
+    depends_on("python@3.8:", type=("build", "run"))
```

Comment:
Says 3.7+ in setup.py. But Spack only supports 3.8+ anyway, so this line isn't actually needed

Reply:
OK, but I would still need 
```python
depends_on("python@3:", type=("build", "run"))
```
right?

Reply:
Nope, Spack doesn't support Python 2 either, so no need to clarify that only 3 is supported

Reply:
OK, sounds good. Removed the python version in https://github.com/spack/spack/pull/42182/commits/9a494497affdcc667149d677464d823cef28f96d